frontend/src/components/Pages/FrontPage/AutoScheduleSection/AutoScheduleSection.tsx

```diff
@@ -56,14 +58,40 @@ const StyledFormCard = styled(FormCard)`
     margin-top: 2px;
 `
 
+const StyledNextAutoMission = styled.div`
+    margin-top: 30px;
+`
+
+const StyledTable = styled(Table)`
+    width: 960px;
+    margin-top: 10px;
+    border-top: 1px solid #dcdcdc;
+`
+
+const skipAutoScheduledMission = async (missionId: string, timeOfDay: string) => {
+    await BackendAPICaller.skipAutoScheduledMission(missionId, timeOfDay)
+}
+
+const allDays = [
+    DaysOfWeek.Monday,
+    DaysOfWeek.Tuesday,
+    DaysOfWeek.Wednesday,
+    DaysOfWeek.Thursday,
+    DaysOfWeek.Friday,
+    DaysOfWeek.Saturday,
+    DaysOfWeek.Sunday,
+]
```

Comment:
Since this is used several places now, it should be moved to AutoScheduleFrequency.ts as be exported